src/main/java/com/prototype/booking/api/referencedata/RoomsInfo.java

```diff
@@ -0,0 +1,31 @@
+package com.prototype.booking.api.referencedata;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor @AllArgsConstructor
+@Entity
+@Table(name ="MEETING_ROOM")
+public class RoomsInfo {
+
+    @Id
+    @Column
+    private String roomId;
```

Comment:
Inside a Room Entity, @Id mostly refers to **roomId**, so **id** should do.

Reply:
Should be ok